content/terraform/v1.12.x/docs/language/state/sensitive-data.mdx

```diff
@@ -13,13 +13,50 @@ When using local state, state is stored in plain-text JSON files.
 
 When using [remote state](/terraform/language/state/remote), state is only ever held in memory when used by Terraform. It may be encrypted at rest, but this depends on the specific remote state backend.
 
-## Ephemeral data
+## Ephemerality
 
--> **Note**: Ephemeral variables, outputs, and resources are available in Terraform v1.10 and later.
```

Comment:
Removing this note because write-only came out in a TF release later, and I feel like each page is a better spot to talk about the specific restrictions.